incubator/mysqlha/templates/NOTES.txt

```diff
@@ -17,3 +17,17 @@ To connect to your database:
 2. Open a connection to one of the MySQL pods
 
     mysql -h {{ template "fullname" . }}-0.{{ template "fullname" . }} -p
+
+{{- if .Values.service.type }}
```

Comment:
does this need to be service.type = nodeport? or add specific documentation for other service types?

Reply:
I added specific `service.Type` logic to this. According to `service.Type` can print different information to the screen